cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -78,14 +80,6 @@ type TestServer struct {
 	EtcdStoragePrefix string                    // storage prefix in etcd
 }
 
-// Logger allows t.Testing and b.Testing to be passed to StartTestServer and StartTestServerOrDie
-type Logger interface {
-	Helper()
-	Errorf(format string, args ...interface{})
-	Fatalf(format string, args ...interface{})
-	Logf(format string, args ...interface{})
-}
-
```

Comment:
I removed this here because it's not enough for `ktesting`, so I would have had to add new methods. It's simpler to just use `testing.TB`.

This is internal, so the effect of the removal is known.

Reply:
Can we split this PR into more commits - this e.g. is unrelated to changes to how we handle cancellation.